config/Version.public.xcconfig

```diff
@@ -1,4 +1,4 @@
 VERSION_SHORT=20.0
 
 // Public long version example: VERSION_LONG=9.9.0.0
-VERSION_LONG=20.0.0.1
+VERSION_LONG=20.0.0.3
```

Comment:
As per PR description, the two versions bump is due to the beta not being merged into `trunk` before the release finalization run.